base_phone/wizard/number_not_found_view.xml

```diff
@@ -39,7 +39,7 @@
                         options="{'dial_button_invisible': True}"/>
                     <field name="current_partner_mobile" widget="phone"
                         options="{'dial_button_invisible': True}"/>
-                    <button name="update_partner" icon="gtk-convert" colspan="2"
+                    <button name="update_partner" class="oe_highlight" colspan="2"
                         string="Update Partner with this Number" type="object"/>
```

Comment:
Wouldn't it be better to put those buttons in `<footer>` ?

Using visibility to show `Update` if a partner has been selected `Create new partner` if no partner was selected.
Plus add a text before `partner_id` saying something like "Select a partner to update it"

I did something like this in:
https://github.com/OCA/stock-logistics-workflow/blob/8.0/stock_shipment_management/wizard/create_shipment_view.xml